server/service/src/dashboard/invoice_count.rs

```diff
@@ -157,6 +157,7 @@ impl InvoiceCountServiceTrait for InvoiceCountService {
         let repo = InvoiceRepository::new(&ctx.connection);
         Ok(repo.count(Some(
             InvoiceFilter::new()
+                .store_id(EqualFilter::equal_to(&ctx.store_id))
                 .r#type(InvoiceRowType::OutboundShipment.equal_to())
                 .status(InvoiceRowStatus::Picked.equal_to()),
```

Comment:
am wondering about our logic here.. the heading on the dashboard is 'Shipments to be picked' but the status is already 'Picked' 🤔

Reply:
Just saw that too after when I went back to dashboard 👀

Reply:
thanks! I see that there's an issue with caching now. Just putting in a fix for that - it's not essential, it just means that when you switch stores the numbers don't update as they should

Reply:
have just snuck in this one 8b5166966 😉